src/fs/main.lua

```diff
@@ -59,6 +59,15 @@ do
     return "/" .. table.concat(k.split_path(path), "/")
   end
 
+  function k.check_absolute(path)
+    checkArg(1, path, "string")
+    if path:sub(1, 1) == "/" then
+      return table.concat(k.split_path(path), "/")
+    else
+      return table.concat(k.split_path(k.current_process().cwd .. "/" .. path), "/")
```

Comment:
check whether there *is* a current process, else this might error sometimes and we don't want that

Reply:
also prepend a `/`

Reply:
> check whether there _is_ a current process

What to do in case there are no current process?